### PR TITLE
Add collectd support

### PR DESCRIPTION
**Pull request by simonpasquier**

### CODE REVIEW

File: influxdb/meta/collectd.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{%- if pillar.influxdb.server is defined %}
+{%- from "influxdb/map.jinja" import server with context %}
+
+{%- if server.admin.get('user', {}).get('enabled', false) and server.http.enabled %}
```
Comment (elemoine) [inline influxdb/meta/collectd.yml]: Suggestion: s/false/False (it makes more sense to me)

Comment (elemoine) [reply influxdb/meta/collectd.yml]: What if `http` is not set on `server`? And what if `enabled` is not set on `http`?

**Review by elemoine — APPROVED** [review]
Minor comments, otherwise looks good to me.

**Review by gthvn1 — APPROVED** [review]
LGTM

File: influxdb/meta/collectd.yml (hunk 1 above)
Comment (simonpasquier) [reply influxdb/meta/collectd.yml]: > What if http is not set on server? And what if enabled is not set on http?

This is ensured by map.jinja.

Comment (elemoine) [reply influxdb/meta/collectd.yml]: Ok

**Pull request merged**